test/commands/generate/models.test.ts

```diff
@@ -161,6 +161,29 @@ describe('models', () => {
       });
   });
 
+  describe('for Kotlin', () => {  
+    test
+      .stderr()
+      .stdout()
+      .command([...generalOptions, 'kotlin', './test/specification.yml', `-o=${ path.resolve(outputDir, './kotlin')}`, '--packageName', 'test.package'])
```

Comment:
```suggestion
      .command([...generalOptions, 'kotlin', './test/specification.yml', `-o=${ path.resolve(outputDir, './kotlin')}`, '--packageName', 'asyncapi.models'])
```

As of version 1.0.0-next.42 `test.package` is marked as an incorrect package name: https://github.com/asyncapi/modelina/pull/1084

You need to adapt the Java test as well.